addOns/tlsdebug/src/main/java/org/zaproxy/zap/extension/tlsdebug/TlsDebugPanel.java

```diff
@@ -92,7 +93,7 @@ private void initialize() {
         JPanel panelContent = new JPanel(new GridBagLayout());
         this.add(panelContent, BorderLayout.NORTH);
 
-        panelContent.setBackground(Color.white);
+        panelContent.setBackground(new Color(UIManager.getColor("TextField.background").getRGB()));
```

Comment:
Why create a new Color? Doesn't getColor return one?

Reply:
It does but the returned `Color` is a [`UIResource`](https://docs.oracle.com/javase/8/docs/api/javax/swing/plaf/UIResource.html) too, so the  `ComponentUI` would handle it as if it was a default colour and could ignore it.